app/src/main/java/org/simple/clinic/scanid/ShortCodeInput.kt

```diff
@@ -0,0 +1,7 @@
+package org.simple.clinic.scanid
+
+data class ShortCodeInput(val shortCodeText: String) {
+  fun isValid(): Boolean {
+    return shortCodeText.length == 7
```

Comment:
We now have this magic number for the length of a BP Passport Short Code duplicated in two places:
- `UuidShortCodeCreatorModule`
- `ShortCodeInput`

The only use case this will change is if we decide to add a new type of BP Passport itself which will require us to rethink how the BP Passport feature works. 

Until then, what do you think about declaring this magic number as a global constant in `Globals.kt`? This only thing defined there now is the screen change animation duration, but I feel this is a good use case for a global constant.

Reply:
Discussed offline: Will be done in the next PR.